experimental/framework/tree-react-api/package.json

```diff
@@ -66,9 +66,7 @@
 		"test:mocha:cjs": "cross-env MOCHA_SPEC=dist/test mocha",
 		"test:mocha:esm": "mocha",
 		"test:mocha:verbose": "cross-env FLUID_TEST_VERBOSE=1 npm run test:mocha",
-		"tsc": "fluid-tsc commonjs --project ./tsconfig.cjs.json && copyfiles -f ../../../common/build/build-common/src/cjs/package.json ./dist",
-		"typetests:gen": "fluid-type-test-generator",
-		"typetests:prepare": "flub typetests --dir . --reset --previous --normalize"
```

Comment:
Yes. Removing this was required to make `pnpm -r run typetests:gen` not crash.

Having no typetests scripts but keeping typeValidation matched other experimental packages I looked at. examples/data-objects/codemirror/package.json is one randomly picked example.

Reply:
Note if you use flub to run type tests prepare over a set of packages (instead of invoking the npm scripts) it will read/edit the typeValidation block, and the disabled flag in there is used to prevent it from reintroducing type tests when you don't want them.

We probably shouldn't have multiple independent ways to disable type tests depending on how scripts are invoked (both a setting in the typeValidation, the presence/absence of a "previous" package alias, and the existence of the package scripts), but we do and having them match seems like our current pattern.